conda_env/cli/main_export.py

```diff
@@ -100,7 +101,7 @@ def execute(args, parser):
         env.add_channels(args.channel)
 
     if args.file is None:
-        print(env.to_yaml())
+        print(json.dumps(env.to_dict())) if args.json else print(env.to_yaml())
```

Comment:
For printing the json there is a function that already exists called `stdout_json`. It will also do a pretty printing of the json, so I think that one should be used. 
The `import json` above can be replaced with `from conda.cli.common import stdout_json`.
And then the `json.dumps(env.to_dict())` can be replaced with `stdout_json(env.to_dict())`

Reply:
Thank you for that, I made those changes!